### PR TITLE
[FROGGER-3] Add .gitignore file

### PR DESCRIPTION
**Pull request by teclamat**
Please review

### CODE REVIEW

**Comment by kseksalk21** [comment]
FAJNE BARDZO

**Review by AdZawadzki — APPROVED** [review]
Good

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (Snowofpoland) [inline .gitignore:1]: I think we should ignore var-option in our project

**Review by Kartlot — APPROVED** [review]
Astonishing

File: .gitignore (hunk 1 above)
Comment (teclamat) [reply .gitignore:1]: okay

**Pull request merged**